### DISCUSSION TITLE
Is there a way to change owner via REST APIs?

### DISCUSSION THREAD

**Discussion by nicokant** [GeoNode/geonode#12140] [Q&A]
Is there some "admin api" or similar to change the owner of a resource?

**Comment by giohappy** [comment]
At the moment the only option is through the users API. See [here](https://docs.geonode.org/en/master/devel/api/usage/index.html#transfer-resources-owned-by-a-user-to-another)

**Reply by ridoo** [reply]
ATM, this option is available on master only, right?

**Reply by giohappy** [reply]
Yes @ridoo
